web/web.go

```diff
@@ -172,7 +172,11 @@ func Run() {
 
 	blogChannel := confAnnouncementsChannel.GetInt()
 	if blogChannel != 0 {
-		go discordblog.RunPoller(common.BotSession, int64(blogChannel), time.Minute)
+		go func() {
+			// Wait for bot session to be ready
+			common.WaitForDiscordReady()
```

Comment:
As previously stated, not convinced this is necessary. Do you have a reproducible series of steps to demonstrate the issue this resolves?